src/test/java/com/marizueva/laboratory/hw4/utils/TestDataProperties.java

```diff
@@ -0,0 +1,26 @@
+package com.marizueva.laboratory.hw4.utils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class TestDataProperties {
```

Comment:
Class should be `final` and has `private` cinstructor

Reply:
fixed